### PR TITLE
Add ICD support (featuremap and commands) to FabricAdmin/FabricBridge samples

### PR DESCRIPTION
**Pull request by andy31415**
Make use of the `is_icd` flag. This builds on top of #34854

### Changes:

- Declare a separate endpoint for ICD bridged devices (this style is terrible ... but unfortunately we have no control over command/attribute lists
- add `TODO` notice that we are re-using version storage, which seems wrong
- Make use of icd flag in feature maps and when deciding what endpoint to present

image

Bit 20 is the feature bit which is ugly ... but ok, 1 << 20 is indeed 1048576 ... also command is indeed 128 (also seems slightly odd ... maybe we expected this to extend/reuse basicinfo)

### CODE REVIEW

File: examples/fabric-bridge-app/fabric-bridge-common/src/BridgedDeviceManager.cpp [hunk 1]
```diff
@@ -127,15 +129,35 @@ constexpr CommandId administratorCommissioningCommands[] = {
...
+    DECLARE_DYNAMIC_CLUSTER(AdministratorCommissioning::Id, AdministratorCommissioningAttrs, ZAP_CLUSTER_MASK(SERVER),
+                            administratorCommissioningCommands, nullptr)
+DECLARE_DYNAMIC_CLUSTER_LIST_END;
+// clang-format on
 
 // Declare Bridged Node endpoint
 DECLARE_DYNAMIC_ENDPOINT(sBridgedNodeEndpoint, bridgedNodeClusters);
+DECLARE_DYNAMIC_ENDPOINT(sIcdBridgedNodeEndpoint, icdBridgedNodeClusters);
+
+// TODO: this is a single version array, however we may have many
```
Comment (tehampson) [inline examples/fabric-bridge-app/fabric-bridge-common/src/BridgedDeviceManager.cpp:159]: What do you mean by this?

Comment (andy31415) [reply examples/fabric-bridge-app/fabric-bridge-common/src/BridgedDeviceManager.cpp:159]: Cluster version is a property of when an attribute within the cluster changes, so I imagine if we have 2 (independent) bridged device A and B ... in the current implementation the cluster version is shared, so if A updates an attribute and we increase a cluster version, the cluster version of B also changes, however that cluster did not change at all.

Unsure how to describe this in a comment, however data sharing for non-const data among bridged devices seems wrong.

**Review by woody-apple — APPROVED** [review]
Approving pending other review

File: examples/fabric-admin/commands/pairing/DeviceSynchronization.h [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#include <app/ReadClient.h>
+#include <controller/CHIPDeviceController.h>
+#include <lib/core/DataModelTypes.h>
+
+#include <memory>
+
+#include "fabric_bridge_service/fabric_bridge_service.pb.h"
+#include "fabric_bridge_service/fabric_bridge_service.rpc.pb.h"
+
+/// Ensures that device data is synchronized to the remove fabric bridge.
```
Comment (yufengwangca) [inline examples/fabric-admin/commands/pairing/DeviceSynchronization.h:29]: This is used to ensures that device data is synchronized to it own fabric bridge, right?

**Review by yufengwangca — APPROVED** [review]

File: examples/fabric-admin/commands/pairing/DeviceSynchronization.h (hunk 2 above)
Comment (andy31415) [reply examples/fabric-admin/commands/pairing/DeviceSynchronization.h]: oops ... remove == remote.
Corrected typo.

Comment (yufengwangca) [reply examples/fabric-admin/commands/pairing/DeviceSynchronization.h]: The remote fabric bridge means the fabric bridge on the other ecosystem?

I think this classed is only used to sync the info to its own fabric-bridge, right?

**Review by tehampson — APPROVED** [review]

**Pull request merged**